Program.cs

```diff
@@ -65,4 +41,12 @@ private static void CreateWithDoubleCheckedLocking()
             var count = lazyVar.Count;
         }
     }
-}
+
+    class Program
+    {
+        static void Main(string[] args)
+        {
+            var summary = BenchmarkRunner.Run<LazyBenchmarks>();
```

Comment:
It is not used but is there because I copy and paste the example code.  The benchmark result along with logs are printed to the console when running from command-line. There are also exporters for users to persist the results to json/markdown/etc. files. I checked their docs but didn't find any place this is used.  The `Summary` class contains various information that might be interesting:
![image](https://user-images.githubusercontent.com/7583839/28801345-ef7ee254-7605-11e7-8512-2176fde8fbe9.png)